### DISCUSSION TITLE
Dreame L10s Pro Ultra Heat: Can't find WiFi access point after rooting

### DISCUSSION THREAD

**Discussion by thekryz** [Hypfer/Valetudo#2158] [Q&A / Support] [answered]
### Are you already using Valetudo?

No, but in the process of starting it

### Your questions

Today, I rooted a brandnew Dreame L10s Pro Ultra Heat. Everything seems to have gone fine with the rooting process. However, after the `fastboot reboot`, when I was supposed to [connect to my robot's access point ](https://valetudo.cloud/pages/installation/dreame.html#fastboot)(beginning of Phase 3), I can't find it. I see lots of WiFi networks from my neighborhood, but none that resembles anything close to my robot. I have tried rebooting the robot many times. What could I be missing or doing wrong?

**Comment by Hypfer** [comment, marked as answer]
![image](https://github.com/user-attachments/assets/e5d4ca10-9140-4804-a887-7a6cfe1b3654)

### Spawn WiFi AP
If you don't see the robot's WiFi AP, try pressing the outer buttons to spawn it again.

When the AP spawns, connect to it and configure WiFi using the valetudo webinterface.

**Reply by thekryz** [reply]
Thank you, that worked :)
